client-app/ui-kit/components/molecules/variations-button/vc-variations-button.vue

```diff
@@ -23,26 +23,35 @@
 
 <script setup lang="ts">
 import { useBreakpoints } from "@vueuse/core";
+import { toRef } from "vue";
 import { useThemeContext } from "@/core/composables";
 import { BREAKPOINTS } from "@/core/constants";
-import { getLinkTarget } from "@/core/utilities/common";
+import type { BrowserTargetType } from "@/core/types";
 
-interface IProps {
-  link?: string;
-  target?: "_blank" | "_self";
-  variationsCount?: number;
-  showLink?: boolean;
+interface IEmits {
+  (event: "linkClick", globalEvent: MouseEvent): void;
 }
 
+defineEmits<IEmits>();
+
 const props = withDefaults(defineProps<IProps>(), {
   link: "",
   variationsCount: 0,
 });
 
+interface IProps {
+  link?: string;
+  target?: BrowserTargetType;
```

Comment:
I think it's better to use local types here.

Reply:
We have many cases when ui-kit uses core types.